src/script/ptl-tool.py

```diff
@@ -43,13 +43,24 @@
 #
 #
 # Merging PR #12345678 into luminous leaving a detached HEAD (i.e. do not update your repo's master branch) and do not label:
+#
 # $ env PTL_TOOL_BASE_PATH=refs/remotes/upstream/ src/script/ptl-tool.py --base luminous --branch HEAD --merge-branch-name luminous --label - 12345678
 # Detaching HEAD onto base: luminous
 # Merging PR #12345678
 # Leaving HEAD detached; no branch anchors your commits
 #
 # Now push to luminous:
 # $ git push upstream HEAD:luminous
+#
+#
+# Fetching PRs by label:
+#
+# $ src/script/ptl-tool.py --base master --branch HEAD --merge-branch master --label - --pr-label wip-pdonnell-testing
+# Detaching HEAD onto base: master
+# Adding labeled PR #18192 to PR list
+# Will merge PRs: [18192]
+# Merging PR #18192
+# Leaving HEAD detached; no branch anchors your commit
```

Comment:
I think we can add this simple example  too:

`src/script/ptl-tool.py --base master --merge-branch <wip-test-branch> --pr-label wip-<name>-testing`

Reply:
I'm trying not to have too many examples. :)